sdk/python/tests/performance_tests.py

```diff
@@ -267,8 +267,8 @@ def load_pipeline_scripts() -> [(Path, str)]:
             fp = Path(path)
 
         assert fp.exists(), f"Cannot find file: {fp.resolve()}"
-
-        pipeline_files_with_name.append((fp, name))
+        for i in range(copies):
```

Comment:
Just to check it can convert to int before use it as range.
```suggestion
        for i in range(int(copies)):
```